client/src/monaco-language-client.ts

```diff
@@ -105,6 +107,11 @@ export class MonacoLanguageClient extends BaseLanguageClient {
         }
     }
 
+    public registerProposedFeatures() {
+        this.registerFeature(new CallHierarchyFeature(this));
```

Comment:
Should `CallHierarchyFeature` be registered? I don't think Monaco supports this feature, does it?

Reply:
Indeed, an additional component will be needed to make it work I guess.

But the SemanticTokensFeature also requires some tweaks to work so... and it seems pretty harmless to register an unsupported feature (especially this one).

What do you propose? Having a method per proposed features?

Reply:
@CGNonofr I was more asking to make sure I hadn't missed it while looking at the code.

I don't think it's the end of the world to register it. It just needs to be made clear that this is unsupported given that Monaco does not support it.

Reply:
Using a comment? or by renaming the method? or just by updating the documentation?